consultation_analyser/batch_calls/batch_calls.py

```diff
@@ -0,0 +1,45 @@
+"""
+Simple class and functions to deploy and monitor AWS Batch jobs. 
+"""
+
+import boto3
+import os
+
+# load env variables
+BATCH_JOB_QUEUE = os.environ.get("BATCH_JOB_QUEUE")
+BATCH_JOB_DEFINITION = os.environ.get("BATCH_JOB_DEFINITION")
+
+
+class BatchJobHandler:
+
+    def submit_job_batch(self, jobName: str, containerOverrides: dict) -> None:
+        client = boto3.client("batch")
+        job = client.submit_job(
+            jobName=jobName,
+            jobQueue=BATCH_JOB_QUEUE,
+            jobDefinition=BATCH_JOB_DEFINITION,
+            containerOverrides=containerOverrides,
+        )
+        return "The job has been submitted successfully with job id: " + job["jobId"]
+
+    def get_job_status(self, jobId: str) -> str:
+        client = boto3.client("batch")
```

Comment:
Would it ever be possible to submit a jobId here that doesn't exist and do we want to check for that (not sure where this will get used so unsure of context)

Reply:
Potentially yes we could. I think as we're not using it yet, we can just catch the exception in our code/make changes when we are (I'll be using this code in one of my upcoming PRs).